### ISSUE TITLE
Mangroves in Protected Areas Data

### ISSUE THREAD

**Issue by klongleywood** [Vizzuality/mangrove-atlas#333]
The 3rd tab (national) in the attached file contains data we can use to populate the mangroves in protected areas widget. It does not include the new basemap as my understanding is it's still under review?

It might be helpful to discuss whether these populations can be updated on the fly in the future with new versions of the WDPA?

[Mangrove_Protection_Calculations_20210430.xlsx](https://github.com/Vizzuality/mangrove-atlas/files/7475421/Mangrove_Protection_Calculations_20210430.xlsx)

**Comment by tamaramegan** [comment]
Hi @lhilarides and @klongleywood, which WDPA version are you using for this calculations? 
The data from this table has data until 2010. Should we use the latest WDPA version from Nov 2021 or do you have a preferred version that matches these calculations?

**Comment by klongleywood** [comment]
Hi @tamaramegan - when you say that this has data through 2010, do you mean 2016? The table I sent should have data through 2016. 

If we're talking on-the-fly calculations, ideally we'd use the most recent version of WDPA data. That said, we have a call later this week to discuss some nuances of this table, including correlating the PA designation year to the mangrove extent, so let's talk more about this on Friday.

**Comment by tamaramegan** [comment]
@klongleywood yes, I meant until 2016!
The question is about which WDPA layer should we be showing on the map. If you have a specific version you want to share or if we should start working with the latest available (Nov 2021)

**Comment by klongleywood** [comment]
Got it! Yes, then I think showing the latest available data makes the most sense.

**Comment by dhakelila** [comment]
Hi Kate, when do you expect the whole data to come? 

I'm trying to evaluate the risks of going ahead with this as I already expressed my preference to have all the materials before Xmas. 

Time will be short. Take into account that you will need to review the widget before the event. 
Also, in terms of development, does it means that we'll work on a version for a single year now and then, upgrade the widget to afford for multiple years? That will mean extra work in our end and I'm afraid we can't afford that. So if we want to have that version ready by February, I'd say that we'll need to keep that version on the current scope.

**Comment by klongleywood** [comment]
@dhakelila and fyi @lhilarides 

When you say the whole data, do you mean the data by year, or just the data for the latest baseline?  If it's the latter, we have been told early January. I'm afraid I don't have a sense of the rest of the calculations, but definitely not in time for a February release. 

I do understand your preference for the data being available pre-Christmas, but if we publish the data that we have as-is, it will not be consistent with future peer-reviewed publications, meaning that we would require another update or we would lose credibility. We are trying to minimize both outcomes by creating a simplified version for just one year.  I hear what you are saying about needing more time if we need to upgrade later on and we can definitely discuss the budget/scope implications for that in the next scoping/budgeting phase. 

I understand too that timing will be tight, but if we can agree on the format/function of the app, which should be quite simple, especially with just one year, it will just be a matter of updating the data when it becomes available. Let's chat more about this when we meet next week.

**Comment by klongleywood** [comment]
Hi @dhakelila @tamaramegan  -- an update on this. Our team met today and we decided that we could run the preliminary analysis before the holiday.  Moving forward, we would keep this widget just showing the most recent baseline/WDPA, and not worry about integrating previous years. 

We may need to update the data table once the published data is released, but that wouldn't involve any change to the widget, just the underlying table.

**Comment by dhakelila** [comment]
Hi @klongleywood, 

We were also discussing this today. 

We thought that what we can do in order to get things done on time is to create ourselves a data sample with the data you already provided and use it to develop the widget, then, provide that sample to you so the only thing you need to do when data is ready is to populate that table and send it back to us so we import to the project. 

If you can provide the data before Xmas it is ok, but we can also proceed the other way if that sounds ok! 

What we do need is approval for the design of the widget for a single year and the agreement that the widget will only show one year in this current scope. 

I'm sending a screenshot with our proposal for that widget. 
![Screenshot 2021-12-13 at 16 18 02](https://user-images.githubusercontent.com/6608294/145838477-c9230d6e-5fdb-482b-bd10-3283881883de.png)

**Comment by klongleywood** [comment]
Hi @dhakelila (and fyi @lhilarides) -- this mock-up looks like it's for restoration, not protection. Can you send one that speaks to the protection stats? It will probably look similar, but I just want to make sure we're on the same page...

**Comment by dhakelila** [comment]
🤦‍♀️ I'm really sorry! I shared the wrong screenshot. But you are right, the one for Protection stats will be very similar. 
![Uploading Screenshot 2021-12-15 at 22.11.10.png…]()

**Comment by dhakelila** [comment]
@klongleywood , @lhilarides 

Clarification about the Protected Areas layer: We have ready the pipeline and we could get a dataset, but Lammert, we prefer if you can provide it. 
If time goes by and you don't get the data, Lammert, then we'll provide one dataset.

**Comment by dhakelila** [comment]
@klongleywood , @lhilarides 

We'll prepare staging next week with the actual data so you can start checking. 
Then, we'll upload new data as soon as we get it.

**Comment by AngelArcones** [comment]
I'm attaching here the example for the new data model for the protected areas widget, as a reference on how new data is needed.
[Mangrove_protected_area.csv](https://github.com/Vizzuality/mangrove-atlas/files/7903701/Mangrove_protected_area.csv)
Also, in order to add the location ID to the new data, I include this file with the ID of all the countries that are currently in the API, and their ISO code. Some countries' names may not perfectly match between the new data and the API info, so it is important to check and correct such cases (or use ISO code if available). 
[Country_location_id.csv](https://github.com/Vizzuality/mangrove-atlas/files/7903700/Country_location_id.csv)

**Comment by dhakelila** [comment]
@klongleywood, @lhilarides, when do you think you will be sending the data? 
Just for us to be ready and upload it asap when we have it.

**Comment by klongleywood** [comment]
- Change text from "Protected mangroves" to "mangroves in protected areas"
- Remove all years except for 2020
- Include total area of mangrove
- Dropdown unit choose between ha and km2

**Comment by aagm** [comment]
Hi sharing here the link to download a cleaned and filtered version from the latest Feb file of wdpa data. take into account that this version is not simplified:
https://storage.googleapis.com/mangrove_atlas/boundaries/processed/wdpa_protected_areas_public.zip

**Comment by lhilarides** [comment]
Hi @aagm , I just checked the WDPA dataset, but it only contains marine protected areas. Since there are quite a lot of mangroves in protected areas that are not marine one, we are also going to need the non-marine ones. Do you have a cleaned / filtered version of the WDPA that includes these? Thanks in advance!

**Comment by aagm** [comment]
Hi, we are folowing the similar rules for WDPA used to generate the version that is uploaded into the platform. That means that We are applying the next filtering option to the main WDPA dataset: 
* We are filtering by Status, we are removing those areas that have the status = proposed.
* We are fitering bt Marine, we are removing Areas with value 0 (Terrestrial), but leaving those that are terrestrial and marine (1), and those that are marine (2).
* We are also filtering by the bbox of the mangrove coverage layer.
We are repairing the geometries but we have not apply any simplification algorithm to the data shared.

If terrestrial is needed we can provide it (we will apply then a different filter related coastal countries to avoid working with unnecesary data), but then numbers calculated from it, will not match at all with the layer we are visualizing in the map.

**Comment by lhilarides** [comment]
Hi @aagm : thanks for explaining! The terrestrial class is also needed as there are many terrestrial protected areas that overlap with the mangrove extent layer. Not including the terrestrial areas in the stats or on the map would mean our statistics for mangrove extent in protected areas per country become meaningless. Just to clarify: "terrestrial" areas can also be located in areas you and I would call coastal. The MARINE attribute of the WDPA layer is very inaccurate in this regard. If it helps: instead of using the bbox you could perhaps intersect the mangrove extent layer with the WDPA layer?

**Comment by aagm** [comment]
Hi! latest wdpa data updated:
this is the[ Link to download](https://storage.googleapis.com/mangrove_atlas/boundaries/processed/wdpa_protected_areas_public.zip):  

* Filtererd by Status, we are removing those areas that have the status = proposed.
* We are also filtering by the geometry of the the mangrove coverage layer.
* We are repairing the geometries but we have not apply any simplification algorithm to the data shared.

**Comment by dhakelila** [comment]
@aagm I think this is still relevant for the WDPA task at hand

**Comment by lhilarides** [comment]
I've updated the "mangroves in protected areas" data for GMW version 3.14. This can be dowloaded here:
https://wetlandsint-my.sharepoint.com/:x:/g/personal/lammert_hilarides_wetlands_org/Ed4f0IFoEN9Ou1qoKT1PM94BYU_1t9wzY-N6QYJQvly8uA?e=Pzoas0

can you upload this to the api @aagm ?
also @klongleywood , here's the long awaited data

**Comment by lhilarides** [comment]
> @lhilarides , you sent us the updated dataset last week and Alicia was working on Friday to get it done. Why has this changed? I don't understand.

The data sent last week came from UNEP-WCMC's Oceans+ platform but was still based on GMW v2. We'd like the numbers between Oceans+ and GMW to correspond, but since WCMC don't have time to update to GMWv3 now we have to calculate this ourselves for the time being. I tried to do this with the WDPA dataset provided above, but this contains >20.000 protected area boundaries, 17.000 of which don't have any mangroves at all (e.g. deserts in Australia) and should therefore not be included.

I then took the latest WDPA (July 2022), applied the filters mentioned above and end up with 3049 protected area boundaries, which I ran the calculations on. The datastructure of the output is the same as the previous
